API/testrunner/devices/artik053.py

```diff
@@ -43,11 +44,16 @@ def initialize(self):
         Flash the device.
         '''
         if self.env['info']['no_flash']:
+            if self.app == 'iotjs':
+                self.buildinfo = self.iotjs_build_info()
             return
 
         tizenrt = self.env['modules']['tizenrt']
         utils.execute(tizenrt['paths']['os'], 'make', ['download', 'ALL'])
 
+        if self.app == 'iotjs':
+            self.buildinfo = self.iotjs_build_info()
+
```

Comment:
All of the device use this `self.buildinfo` in case of `iotjs` application but only in the `skiplist.py`. There are a lot of code duplication. I would rather create a common `iotjs_build_info` function in the `RemoteDevice` class which return with three values: `builtin_modules`, `builtin_features`, `stability`. Then you can call it from the `_skip_iotjs_test` directly.